Sources/TuistGraph/DependenciesGraph/ThirdPartyDependency.swift

```diff
@@ -2,20 +2,67 @@ import Foundation
 import TSCBasic
 
 // A enum containing information about third party dependency.
-public enum ThirdPartyDependency: Hashable, Equatable, Codable {
+public enum ThirdPartyDependency: Hashable, Codable {
+    /// A dependency that is imported as source code.
```

Comment:
I don't think `sources` is a particularly good name. I'd call it just `spm` since it is currently only used for that.